packages/pigeon/example/app/pigeons/messages.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(PigeonOptions(
+  dartOut: 'lib/src/messages.g.dart',
+  cppOptions: CppOptions(namespace: 'pigeon_example'),
+  cppHeaderOut: 'windows/runner/messages.g.h',
+  cppSourceOut: 'windows/runner/messages.g.cpp',
+  kotlinOut:
+      'android/app/src/main/kotlin/dev/flutter/pigeon_example_app/Messages.g.kt',
+  // This file is also used by the macOS project.
```

Comment:
Do you think having multiple outputs is something anyone would be interested in?

Reply:
Now that we have `sharedDarwinSource` for plugins (which allows macOS and iOS to share the same source directory) I think it's less likely. The use cases are now:
- apps, as with this case, and
- plugins that support macOS and iOS, with the pigeon API, but with completely unrelated implementations.

I think it's fine to just wait and see if we get any real-world requests for it.